### PR TITLE
FEATURE-7: performed all current tasks

### PR DESCRIPTION
**Pull request by diledsys**
HomeWork_diledsys_Car_task2,3,4

### CODE REVIEW

File: src/main/java/com/java_essential/diledsys/Classwork/Course/Course.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+
+public class Course {
+   private long id;
+   private String courseName;
+   private String description;
+   private Student[] student;
+
+    public Student[] getStudent() {
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_essential/diledsys/Classwork/Course/Course.java]: Getters and setters should be located at the same place

File: src/main/java/com/java_essential/diledsys/Classwork/Student/Student.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }//dfgdssdg
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_essential/diledsys/Classwork/Student/Student.java]: delete unnecessary comment please

File: src/main/java/com/java_essential/diledsys/homeWork/Computer/Computer.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.java_essential.diledsys.homeWork.Computer;
+
+public class Computer {
+    public static void main(String[] args) {
+
+        Object[] computers = new Object[5];
+
+        int i = 0;
+        do {
+            ;
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_essential/diledsys/homeWork/Computer/Computer.java]: delete this symbol

File: src/main/java/com/java_essential/diledsys/homeWork/task4_Car/Main.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Car mazda = new Car(2017);
+        Car renault = new Car(2014, 150.0);
+        Car mersedes = new Car(2020, 220, 2000);
+        Car kya = new Car(2017, 180, 1700, "green");
+        //Car zm =new Car("red",180,200,2000);
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_essential/diledsys/homeWork/task4_Car/Main.java]: delete this duplicate object

File: src/main/java/com/java_essential/diledsys/myExperiments/Singl/Singl.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package com.java_essential.diledsys.myExperiments.Singl;
+
+public class Singl {
```
Comment (yuriy-tsarenko) [inline src/main/java/com/java_essential/diledsys/myExperiments/Singl/Singl.java:3]: NOTE: constructor with "private" modifier have preferred usage in utility classes,
in this case you should add "final" modifier to class "Singl".
It means that extending of class "Singl" is forbidden.
We'll learn more details about it during our future lessons.

**Review by yuriy-tsarenko — CHANGES_REQUESTED** [review]
Fix code according to my comments

File: src/main/java/com/java_essential/diledsys/myExperiments/Singl/Singl.java (hunk 5 above)
Comment (diledsys) [reply src/main/java/com/java_essential/diledsys/myExperiments/Singl/Singl.java:3]: дддрол

Comment (diledsys) [reply src/main/java/com/java_essential/diledsys/myExperiments/Singl/Singl.java:3]: да final тут нужен, это просто тест, я еще делал тест с класом на приоритеты, тоже интиресные моменты , со статичиским блоком и инициализирующим

**Review by yuriy-tsarenko — APPROVED** [review]

**Pull request merged**